packages/stdlib/test.pony

```diff
@@ -62,6 +63,12 @@ actor Main is TestList
     logger.Main.make().tests(test)
     net.Main.make().tests(test)
     options.Main.make().tests(test)
+
+    ifdef not windows then
```

Comment:
I don't want to be too annoying here, but since we use `ifdef posix` everywhere in the implementation, it probably makes sense to use `ifdef posix` here too instead of `ifdef not windows`.